gradle.properties

```diff
@@ -1,10 +1,10 @@
 group=org.sonarsource.scanner.gradle
-version=4.2-SNAPSHOT
+version=4.2.0.3127
```

Comment:
indeed, but the release I triggered was pointing to 4.2-SNAPSHOT, so it generated 4.2.0.3127; honestly, I don't know if it is worth to re-do it to have 4.1.1 instead of 4.2.0